### DISCUSSION TITLE
Support for next 13?

### DISCUSSION THREAD

**Discussion by sammaji** [blitz-js/blitz#4210] [Q&A]
Will we get support for next 13, app router, server components etc?

**Comment by tordans** [comment]
@samyabrata-maji  AFAIK it should all work.

Here are a few infos I know of…
* https://github.com/blitz-js/blitz/pull/4079 merged a lot of changes to get the app directory working
* https://github.com/blitz-js/blitz/issues/4070 was an issue that tracked required changes
* I just found some docs on the topic at https://blitzjs.com/docs/usage-next-13#app-directory
* https://github.com/jhonnymichel/fitx/blob/app-dir/src/days/components/DaySummary.tsx is a demo app (I read about it on the discord) which might showcase how to use the app router. One think I am not sure about if it where possible to move the components in the new app-dir (in this example, they are still part of the src folder)

**Reply by sammaji** [reply]
What I meant was, I wanted the generated boilerplate code to be using the app directory and not the pages directory. Is there a way to do this?

**Reply by tordans** [reply]
AFAIK no. But maybe it is time to create a PR to switch the generator to the app directory. Maybe ask siddhsuresh in Discord if this would be merged.

**Reply by sammaji** [reply]
Sure thanks!
